site/content/en/blog/releases/1.39.0.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "1.39.0 - New GenAI and Custom Controller Examples and Counters and Lists Values Available on Allocation Constraints"
+linktitle: "1.39.0"
+date: "2024-03-12"
+---
+
+This is the 1.39.0 release of Agones.
+
+In this release, we’ve made some important changes and introduced exciting new features, including an example for integrating GenAI, improved GameServer allocation results for Counters and Lists, and a new example custom GameServer controller.
+
+**Breaking Changes:**
+
+- **C++ gRPC Setup**: This is a breaking change with our dependency management for C++ and gRPC. Our CMake file no longer automatically imports and compiles gRPC, as it was getting increasingly fragile. Now, to install the C++ SDK, you will need to install gRPC manually before installing the C++ SDK.
+
+- **GKE Terraform - Agones-Metrics nodepools**: With our latest tweak in GKE Terraform configuration, the `agones-metrics` nodepool is disabled by default, as not everyone will want it. If you do need it for Prometheus and Grafana? Just turn it on and you’re set.
+
+
+**Key highlights of this release include:**
+
+- **GenAI and Agones**:  We've added a new [example GenAI game server](https://github.com/googleforgames/agones/tree/release-1.39.0/examples/simple-genai-server) that shows how to integrate Generative AI backends into your Agones game server. 
```

Comment:
This link needs the data proofer ignore attribute since the link won't exist until after this PR gets submitted.

Reply:
`<a href="https://github.com/googleforgames/agones/tree/release-1.39.0/examples/simple-genai-server" data-proofer-ignore>example GenAI game server</a>`

Reply:
Done!